packages/core/src/Box/README.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: 'Box'
+type: 'component'
+slug: /components/box/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/Box'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-box--default'
+---
+
+Box is the most abstract layout component that allows to apply `margin` and `padding` properties according to the [spacing system]. By default, it renders a `div` element.
+
+The Box component is useful because it helps with several common cases:
+
+- Create a layout that uses Forma36 [spacing system]
+- Compose new component and allow to override element using the `as` property.
+
+## Import
+
+```js static=true
+import { Box } from '@contentful/f36-components';
+```
+
+## Usage
+
+### Margins & Paddings
+
+Margins and paddings for Box component are based on our [spacing system].
+
+Here is an overview of spacings that are available:
+
+- `spacing2Xs`
+- `spacingXs`
+- `spacingXs`
+- `spacingS`
+- `spacingM`
+- `spacingL`
+- `spacingXl`
+- `spacing2Xl`
+- `spacing3Xl`
+- `spacing4Xl`
+
+```jsx
+() => {
+  const SmallWhiteBox = () => (
+    <Box
+      as="span"
+      style={{
+        width: '30px',
+        height: '30px',
+        backgroundColor: tokens.colorWhite,
+      }}
+    />
+  );
+
+  return (
+    <div style={{ display: 'flex', alignItems: 'center' }}>
+      <Box
+        padding="spacingXs"
+        marginRight="spacingM"
+        style={{
+          backgroundColor: tokens.colorRedBase,
+        }}
+      >
+        <SmallWhiteBox />
+      </Box>
+      <Box
+        padding="spacingXs"
+        marginRight="spacingM"
+        style={{
+          backgroundColor: tokens.colorBlueBase,
+        }}
+      >
+        <SmallWhiteBox />
+      </Box>
+    </div>
+  );
+};
+```
+
+### as property
+
+You can use the `as` prop to change the element render, just like styled-components.
```

Comment:
I'd like if we could come up with our own explanation here, maybe just two examples of rendering a span and a div or something?

Reply:
There's an example using it as "span" below.

Reply:
That's true :) I was thinking maybe in text instead of referring to styled-components. Or maybe a link to them?

Reply:
ah, I see... yeah, I'll add our own explanation here

Reply:
updated!